data/human/hails.txt

```diff
@@ -1670,6 +1670,47 @@ phrase "friendly civilian"
 		"The next event will be great!"
 		"That's real entertainment!"
 
+phrase "friendly civilian"
+	word
+		"I"
+		"Me and my friend"
+	word
+		" "
+	word
+		"had a job"
+		"used to spend time on"
+		"worked for years on"
+		"worked in the business of"
+	word
+		" "
+	word
+		"creating pixel art"
+		"creating computer games"
+		"creating 3D computer models"
+		"creating websites on the worlds wide web"
+		"cleaning corridors in an university"
+		"delivering furniture to peoples houses"
+		"documenting complex processes"
+		"farming in the dirt belt"
```

Comment:
```suggestion
		"farming in the Dirt Belt"
```
_Is_ Dirt Belt supposed to be capitalized? I actually don't know, but I think it is. Feel free to resolve if I'm wrong.